data_dir/large_collections.yaml

```diff
@@ -55,17 +55,18 @@ columnspec:
 # 1448.1546878700494
 # >>> math.sqrt(1 * 1024 * 1024)
 # 1024.0
+# gaussian(min..max,mean,stdev)
```

Comment:
@Mark-Gurevich are all the comments above still relevant?
If not, please add your comments explaining the size and population you used.

Reply:
https://github.com/scylladb/qa-tasks/issues/894#issuecomment-1713449685

`if gaussian, min = 6k, max = 12k, mid = 9k, diff = 13%`
This calculation gives us that most of the rows do not exceed the threshold, but only some do as you requested.
Anyway we still filter the warnings mentioned above.

Reply:
Your answer doesn't answer what I asked.
I see some calculations above in the comments, but the numbers you're using are different from them.
So, I asked if these are still relevant.

And regarding to what you've answered about "size: gaussian(6000..12000,9000,13)", it would be good to add it in the comment, so the next person read it, will understand why you chose these numbers.